packages/botonic-plugin-contentful/src/cms/contents.ts

```diff
@@ -488,8 +488,8 @@ export class Handoff extends TopContent {
   constructor(
     readonly common: CommonFields,
     readonly onFinish: OnFinish,
-    readonly message?: string,
-    readonly failMessage?: string,
+    readonly message?: Text | string,
+    readonly failMessage?: Text | string,
```

Comment:
In this line you say that it can be a string but I don't see in the rest of the code that a string is allowed for the message and failMessage fields. 
In the HandoffBuilder these fields should be allowed to be Text or string, right?

Reply:
These messages come from the handoff object. In the handoff delivery class when the handoff messages are received I check if they are a string, otherwise I put them as references.